a3fe/run/leg.py

```diff
@@ -717,13 +719,13 @@ def run_ensemble_equilibration(
         else:  # Free leg
             return pre_equilibrated_system
 
-    def setup_stages(
+    def setup_systemprep_stages(
```

Comment:
Please reverse name to setup_stages - sorry for being unclear. Please just name "config" "system_prep_config" to make it clearer what it is.